Tests/Functionnal/app/config/config_base.yml

```diff
@@ -121,8 +121,10 @@ security:
             switch_user: ~
         main:
             pattern: ^/
+#            anonymous: true
             http_basic: ~
     access_control:
+        - { path: ^/, roles: IS_AUTHENTICATED_ANONYMOUSLY }
```

Comment:
The 2 lines must be inverted, because the first line will match all the requests, as it's written the admin wont be protected:

> For each incoming request, Symfony checks each `access_control` entry to find one that matches the current request. As soon as it finds a matching `access_control` entry, it stops - only the **first** matching `access_control` is used to enforce access.

Source: https://symfony.com/doc/2.8/security/access_control.html